app/src/main/java/com/lastaoutdoor/lasta/ui/screen/discover/DiscoverScreen.kt

```diff
@@ -114,31 +116,37 @@ fun DiscoverScreen(
       }
 
   if (screen == DiscoverDisplayType.LIST) {
-    LazyColumn(
+    Column(
         modifier =
             Modifier.testTag("discoveryScreen").background(MaterialTheme.colorScheme.background)) {
-          item {
-            HeaderComposable(
-                screen,
-                range,
-                selectedLocality,
-                setScreen,
-                { isRangePopup = true },
-                navigateToFilter,
-                orderingBy,
-                updateOrderingBy,
-                weather)
-          }
+          HeaderComposable(
+              screen,
+              range,
+              selectedLocality,
+              setScreen,
+              { isRangePopup = true },
+              navigateToFilter,
+              orderingBy,
+              updateOrderingBy,
+              weather)
 
-          item {
-            Spacer(modifier = Modifier.height(8.dp))
-            ActivitiesDisplay(
-                activities,
-                centerPoint,
-                favorites,
-                changeActivityToDisplay,
-                flipFavorite,
-                navigateToMoreInfo)
+          Spacer(modifier = Modifier.height(8.dp))
+          if (isLoading) {
+            LoadingAnim(width = 35, tag = "LoadingBarDiscover")
+          } else if (activities.isEmpty()) {
+            /* TODO */
+          } else {
+            LazyColumn {
```

Comment:
Great job! It's indeed better to have the lazy column being just the list of activities and have a fixed header, wouldn't have thought of it myself !